clar2wasm/src/tools.rs

```diff
@@ -341,8 +341,37 @@ pub fn interpret(snippet: &str) -> Result<Option<Value>, Error> {
     interpret_at(snippet, StacksEpochId::latest(), ClarityVersion::latest())
 }
 
+struct TestConfig;
+
+impl TestConfig {
+    /// Select a Clarity version based on enabled features.
+    pub fn clarity_version() -> ClarityVersion {
+        match () {
+            _ if cfg!(feature = "test-clarity-v1") => ClarityVersion::Clarity1,
+            _ if cfg!(feature = "test-clarity-v2") => ClarityVersion::Clarity2,
+            _ if cfg!(feature = "test-clarity-v3") || cfg!(feature = "test-clarity-latest") => {
+                ClarityVersion::Clarity3
+            }
+            _ => ClarityVersion::latest(),
```

Comment:
Given that we have `ClarityVersion::latest()`, it seems strange to have `test-clarity-latest` resolving to something different. Why not `_ if cfg!(feature = "test-clarity-latest") => ClarityVersion::latest()` (or removing it from the check and rely on `_` to get latest)?

Reply:
Removed `feature = "test-clarity-latest"`. It seems bloated to have that indeed.